spellbook/macros/optimize_tables.sql

```diff
@@ -56,10 +56,22 @@ OPTIMIZE nft.burns;
 OPTIMIZE nft.fees;
 {% endset %}
 
+{% set uniswap_v1_ethereum_trades %}
+OPTIMIZE uniswap_v1_ethereum.trades;
+{% endset %}
+
+{% set uniswap_ethereum_trades %}
+OPTIMIZE uniswap_ethereum.trades;
+{% endset %}
+
 {% set uniswap_trades %}
 OPTIMIZE uniswap.trades;
 {% endset %}
 
+{% set dex_trades %}
+OPTIMIZE dex.trades;
```

Comment:
are we sure this should be a table? should it be a view of unioned tables instead?

Reply:
i'm following the standard set by nft trades, which is also an incremental table and included in the optimize statements. maybe an assumption that it was discussed and agreed upon. we can explore unions and default view, would be curious to see performance difference. either way, i assume we should match between dex and nft

Reply:
set back to views for everything outside of the first layer reading from decoded tables, which are left incremental